code/modules/reagents/chems/chems_painkillers.dm

```diff
@@ -25,37 +34,76 @@
 
 	M.add_chemical_effect(CE_PAINKILLER, (pain_power * effectiveness))
 
-	if(dose > 0.5 * overdose)
-		M.add_chemical_effect(CE_SLOWDOWN, 1)
-		if(prob(1))
-			SET_STATUS_MAX(M, STAT_SLUR, 10)
-	if(dose > 0.75 * overdose)
-		M.add_chemical_effect(CE_SLOWDOWN, 1)
-		if(prob(5))
-			SET_STATUS_MAX(M, STAT_SLUR, 20)
-	if(dose > overdose)
-		M.add_chemical_effect(CE_SLOWDOWN, 1)
-		SET_STATUS_MAX(M, STAT_SLUR, 30)
-		if(prob(1))
-			SET_STATUS_MAX(M, STAT_WEAK, 2)
-			SET_STATUS_MAX(M, STAT_DROWSY, 5)
+	if(!narcotic)
```

Comment:
Not sure what the purpose of this is. Might have been inherited from the Hearth file.

Reply:
None of the side effects will trigger if it's not a narcotic painkiller.